application/src/main/java/org/togetherjava/tjbot/commands/BotCommandAdapter.java

```diff
@@ -97,6 +98,13 @@ public void onSelectionMenu(SelectMenuInteractionEvent event, List<String> args)
         // Adapter does not react by default, subclasses may change this behavior
     }
 
+    @SuppressWarnings("NoopMethodInAbstractClass")
+    @Override
+    public void onModal(final ModalInteractionEvent event, final List<String> args) {
```

Comment:
remove the finals to make it more in line with other methods 
```suggestion
    public void onModal(ModalInteractionEvent event, List<String> args) {
```

Reply:
fixed